utility/color_util.go

```diff
@@ -45,6 +49,21 @@ func Red(value string) string {
 
 // Error is the function to handler all error in the Cli
 func Error(msg string, args ...interface{}) {
+	githubTag := &latest.GithubTag{
```

Comment:
If there are no cyclic imports, can we replace this bit with common.VersionCheck ? ˀ

Reply:
Yes, I have replaced it